spec/services/jbpm_process_service_spec.rb

```diff
@@ -64,20 +63,39 @@
     end
   end
 
-  context 'when kie service raise exception' do
-    before do
-      allow(jbpm).to receive(:start_process).and_raise(kie_ex)
-      allow(subject).to receive(:enhance_groups)
+  describe 'kie service raise exception' do
+    context 'when start_process' do
+      before do
+        allow(jbpm).to receive(:start_process).and_raise(kie_ex)
+        allow(subject).to receive(:enhance_groups)
```

Comment:
I know you just moved this so I don't think we need to change it here, but stubbing out a private method is something I'd like to shy away from. It tells me that either this class is trying to do too much and it's complicated to test, or that there's some functionality that isn't really that important for this test and it could be broken out somewhere else. Just something to keep in mind.

Reply:
👍